com.ozmartians.VidCutter.json

```diff
@@ -0,0 +1,302 @@
+{
+    "app-id": "com.ozmartians.VidCutter",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "vidcutter",
+    "rename-appdata-file": "com.ozmartians.vidcutter.appdata.xml",
+    "rename-desktop-file": "com.ozmartians.vidcutter.desktop",
```

Comment:
Do you plan on renaming these upstream btw? These are the recommendations from the specs and it is best when flatpaks match normal packages.

Reply:
Oh looking upstream these are already renamed, so you should just remove these lines.